lib/iris/fileformats/pp_rules.py

```diff
@@ -1008,14 +1018,48 @@ def _all_other_rules(f):
                 bounds=f.x_bounds),
              1))
 
+    if (len(f.lbcode) == 5 and f.lbcode[-1] == 3 and f.lbcode.iy == 23):
+        epoch_days_unit = cf_units.Unit('days since 0000-01-01 00:00:00',
+                                        calendar=cf_units.CALENDAR_360_DAY)
+        t1_epoch_days = epoch_days_unit.date2num(f.t1)
+        t2_epoch_days = epoch_days_unit.date2num(f.t2)
+        # The end time is exclusive, not inclusive.
+        dim_coords_and_dims.append(
+            (DimCoord(
+                np.linspace(t1_epoch_days, t2_epoch_days, f.lbrow,
+                            endpoint=False),
+                standard_name='time',
+                units=epoch_days_unit,
+                bounds=f.y_bounds),
+             0))
+
     # Site number (--> scalar coordinate)
-    if (len(f.lbcode) == 5 and f.lbcode[-1] == 1 and f.lbcode.ix == 13 and
-            f.bdx != 0):
+    if (len(f.lbcode) == 5 and f.lbcode[-1] == 1 and f.lbcode.ix == 13):
         dim_coords_and_dims.append(
-            (DimCoord.from_regular(f.bzx, f.bdx, f.lbnpt,
-                                   long_name='site_number', units='1'),
+            (DimCoord(
+                f.x,
+                long_name='site_number',
+                units='1'),
```

Comment:
I will remove this change, I mostly added it to highlight the non-conformity to the docs.